Resources/ArmTemplates/template.json

```diff
@@ -349,21 +349,29 @@
                                 "name": "Configuration__CourseSearchClient__CourseSearchSvc__SearchPageSize",
                                 "value": "20"
                             },
-                            {
-                                "name": "Configuration__CourseSearchClient__CourseSearchSvc__AttendanceModes",
-                                "value": "AM1,AM2,AM3,AM4,AM5,AM6,AM7,AM8,AM9"
-                            },
                             {
                                 "name": "Configuration__CourseSearchClient__CourseSearchSvc__RequestTimeOutSeconds",
                                 "value": "10"
                             },
                             {
-                              "name": "Configuration__CourseSearchClient__CourseSearchSvc__MaxReceivedMessageSize",
-                              "value": "2147483647"
+                                "name": "Configuration__CourseSearchClient__CourseSearchSvc__TransintErrorsNumberOfRetries",
```

Comment:
Is this meant to be TransientErrors rather than Transint?

Reply:
Corrected this typo in the Client sdk nuget package which is where these settings are used (and need to match)